bindings/matlab/iio-wrapper.h

```diff
@@ -1,4 +1,56 @@
 #ifndef MATLAB_LOADLIBRARY
 #define MATLAB_LOADLIBRARY
 #include <iio.h>
+
+#ifdef _WIN32
+#   ifdef LIBIIO_EXPORTS
+#	define __api __declspec(dllexport)
+#   else
+#	define __api __declspec(dllimport)
+#   endif
+#elif __GNUC__ >= 4 && !defined(MATLAB_MEX_FILE) && !defined(MATLAB_LOADLIBRARY)
+#   define __api __attribute__((visibility ("default")))
```

Comment:
Actually MATLAB_LOADLIBRARY is for when Matlab reads iio-wrapper.h then iio.h, so that it doesn't choke on __api, since it does not handle GCC's __attribute__ stuff. In that case, __api should not be defined...

Ok, what about this:
* in iio-wrapper.h, add this:
`#ifndef __api  #define __api  #endif`
* Don't include bindings/matlab/iio-wrapper.h from scan.c; but include bindings/matlab/iio-wrapper.h from iio-private.h, after __api is defined.